index.html

```diff
@@ -62,27 +63,27 @@ <h1 class="head__title fz-40 fw-700 lh-52">
       <a href="#works" class="head__link link fz-16 fw-500 lh-24 clr-p-400">My Projects</a>
       <ul class="head__icons ul flex">
         <li class="head__icons__li">
-          <a href="https://twitter.com/SilviaTofana" class="head__icons__link link"><img src="icons/twitter.png" alt="twitter icon" /></a>
+          <a href="https://twitter.com/silviatofana" class="head__icons__link link"><img src="images/twitter.png" alt="twitter icon" /></a>
         </li>
         <li class="head__icons__li">
-          <a href="www.linkedin.com/in/silvia-tofana-10b852186/" class="head__icons__link link"><img src="icons/Linkedin.png" alt="linkedin icon" /></a>
+          <a href="https://www.linkedin.com/in/silviatofana/" class="head__icons__link link"><img src="images/Linkedin.svg" alt="linkedin icon" /></a>
         </li>
         <li class="head__icons__li">
-          <a href="#" class="head__icons__link link"><img src="icons/m.png" alt="M icon" /></a>
+          <a href="#" class="head__icons__link link"><img src="images/M.png" alt="M icon" /></a>
         </li>
         <li class="head__icons__li">
-          <a href="https://github.com/silviatofana" class="head__icons__link link"><img src="icons/github.png" alt="github icon" /></a>
+          <a href="https://github.com/silviatofana" class="head__icons__link link"><img src="images/github.svg" alt="github icon" /></a>
         </li>
         <li class="head__icons__li">
-          <a href="#" class="head__icons__link link"><img src="icons/peace.png" alt="hand icon" /></a>
+          <a href="#" class="head__icons__link link"><img src="images/hand.svg" alt="hand icon" /></a>
         </li>
       </ul>
     </section>
 
     <section class="works" id="works">
```

Comment:
- [ ] [optional] I am marking this requirement as optional because it was missed by the previous reviewer, But I strongly suggest considering it.
 - You should update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads. (You may want to delete the code in the HTML file for the project section and create it dynamically using javascript).